projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/judicial-user/write-judicial-user-field.component.spec.ts

```diff
@@ -104,7 +124,9 @@ describe('WriteJudicialUserFieldComponent', () => {
       schemas: [CUSTOM_ELEMENTS_SCHEMA],
       providers: [
         { provide: ActivatedRoute, useValue: activatedRoute },
```

Comment:
is ActivatedRoute still needed?

Reply:
I have removed the ActivatedRoute as it is no longer required. Thank you.